README.md

```diff
@@ -113,51 +167,94 @@ You need to set up these environment variables:
+      "**/+([^:]):+([^:])/**",
+      "**/@(test|tests|spec|specs)/**",
+      "**/@(test.?*|?*.test.?*|?*.tests.?*)",
+      "**/@(spec.?*|?*.spec.?*|?*.specs.?*)",
+      "**/@(app-test.?*|?*.app-test.?*|?*.app-tests.?*)",
+      "**/@(app-spec.?*|?*.app-spec.?*|?*.app-specs.?*)"
+    ]
+  },
+  "output": "./.coverage"
 }
 ```
+Details : 
+
+ - Allows / Disallow is coded with the following order `include`, `exclude.general`, `exclude.(client|server)`, it is used before both instrumentation and before coverage report creation.
+ - The glob syntax can be found [here](https://github.com/isaacs/node-glob#glob-primer).
+ -  To create your custom config file, run the project with `COVERAGE_VERBOSE=1` env variable and use logs to see which filenames were hooked or hidden. PR welcome.
+- The output folder needs to starts with a dot to exclude that folder from Meteor build.
 
-The minimatch syntax of this file can be found [here](http://hgbook.red-bean.com/read/file-names-and-pattern-matching.html#id381184).
+### Flow router issue
 
-To create your custom config file, run the project with `COVERAGE_VERBOSE=1` env variable and use logs to see which filenames were hooked or hidden. PR welcome.
+If you are using flow-router, there are [an issue](https://github.com/kadirahq/flow-router/pull/615) that prevents tests to succeed.  
+This workaround replaces flow-router with a patched version which ignores that error: 
+```bash
+cd packages 
+git submodule add https://github.com/serut/flow-router
+```
+
+## My files are missing from my app coverage report
+
+If you have **internal packages** inside your app and you want to get their **server side** coverage. Open the file `.meteor/packages` and move the line `lmieulet:meteor-coverage` to be above these packages.
+
+### Istanbul html colors legend
+
+- Pink: statement not covered
+- Orange: function not covered
+- Yellow: branch not covered
+- [I] and [E] in front of if-else statements: if or else not covered respectively
+- Branch coverage display only kicks in if one or more but not all branches have been taken (if none of the branches were taken the statement coverage will show you that unambiguously)
+
+## Ignore code from coverage with annotation
+
+For example, if you code an `if` block without `else`, istanbul marks the `else` branch as not covered (although it doesn't exist), decreasing your code coverage, which is **false**.
+
+The syntax can be found there:  
+https://github.com/gotwarlost/istanbul/blob/master/ignoring-code-for-coverage.md
+
+## Meteor ignored folders and files
+
+- hidden folders like .npm, .coverage or .meteor.
+- special folders like node_modules.
+- all meteor packages (bundled and/or manually installed ones) like meteor/underscore, meteor/accounts-password or aldeed:simple-schema.
+- all tests file(s) like *.(specs?|tests?|app-specs?|app-tests?).* or (specs?|tests?|app-specs?|app-tests?).*, and all folder(s) like (specs?|tests?|app-specs?|app-tests?).
+
```

Comment:
Right now, I think the special folder `node_modules` is not always excluded due to the issue explained in the glob patterns comment.

At the fourth item of the list, the first and last asterisks of the first glob pattern are interpreted by GitHub as styling that portion of text as italic (and thus not shown): you can style those portions as code (`*.(specs?|tests?|app-specs?|app-tests?).*`) or escape those asterisks (\*.(specs?|tests?|app-specs?|app-tests?).\*), whatever you prefer.